### DISCUSSION TITLE
Melting raw ore blocks

### DISCUSSION THREAD

**Discussion by avescarelo** [Roadhog360/Et-Futurum-Requiem#391] [Ideas]
At the moment, raw blocks of copper, iron and gold can only be crafted into 9 ore items on the workbench. 

I suggest adding the ability to immediately melt 1 raw block into 9 ingots.

![image](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/26678843/d921c3a8-fb42-4b41-95da-e67abfef3d16)

**Comment by GregoriusT** [comment]
Furnace Recipes do not allow for using more than one smelt worth of Fuel, so this is not really an  option there.

**Comment by Roadhog360** [comment]
This is not vanilla so it will not be added, in addition to the fact that this would be op because you could smell ores with nine times efficiency for free
